guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -495,6 +502,9 @@ A CHANGELOG entry should summarize what was changed and should end with the auth
 Your name can be added directly after the last word if there are no code
 examples or multiple paragraphs. Otherwise, it's best to make a new paragraph.
 
+### Ignoring files created by your editor/IDE
```

Comment:
@rafaelfranca I've also added this section. I was not aware that git had a global ignore file when I opened that PR to change Rails `.gitignore` and ended being quite embarrassed about having wasted your time like that 😅